Dockerfile

```diff
@@ -0,0 +1,96 @@
+# syntax = docker/dockerfile:1
+ARG RUBY_VERSION=3.4.1
```

Comment:
I think all the versions (Ruby, node, yarn) here need to be synced with what we have on local, so they need to come from something like a global var or `.ruby-version`, `.node-version` (not sure what it would be for yarn - the version is just a string in the package.json atm).

Reply:
I have done a couple of attempts to pull the version from .ruby-version, but couldn't get it working from within the Dockerfile... Args don't seem to support shell expansions.

But most probably there is a kamal way to dynamically set build args:
https://kamal-deploy.org/docs/configuration/builder-examples/#configuring-build-args-for-new-images